BUILD

```diff
@@ -74,4 +76,8 @@ haskell_library(
 sh_binary(
     name = "spark-submit",
     srcs = ["@spark//:spark-submit"],
+    data = ["//apps/deltalake-glow:genotypes.vcf",
+    	    "//apps/deltalake-glow:continuous-phenotypes.csv",
+	    "//apps/deltalake-glow:covariates.csv"
+	    ],
```

Comment:
Would it be possible to move the dependencies and the datafiles in this module to the places where they are needed? This would be in `apps/deltalake-glow/BUILD.bazel`. Otherwise, all of the applications using `spark-submit` and `sparkle-lib` would also have to depend indirectly on them.

Reply:
Yes, great point! 
Bazel complains when I tried to created any `sh_binary` in `apps/deltalake-glow/BUILD.bazel`, so I created another `sh_binary` in `BUILD` called `spark-submit-with-data` to include the datafiles and make it independent. Does it sound good to you?

Reply:
I think it is fine for now. Thanks!